persistence/attributes.go

```diff
@@ -336,7 +336,7 @@ func AttributesToEnvvarMap(attributes []Attribute, envvars map[string]string, pr
 	writePrefix("ARCH", cutil.ArchString())
 
 	// Override with the built-in properties
-	externalPol := externalpolicy.CreateNodeBuiltInPolicy(false)
+	externalPol := externalpolicy.CreateNodeBuiltInPolicy(false, true, nil)
```

Comment:
HARDWAREID is missing from the containers, I think you need to get the node policy from the DB and pass it in and I'm seeing several hardware ids in the log again.

Reply:
Fixed. HARDWAREID is now in the containers

Reply:
Fixed